app/components/candidate_interface/degree_new_review_component.rb

```diff
@@ -54,12 +55,28 @@ def abbreviate_degree(name)
       Hesa::DegreeType.find_by_name(name)&.abbreviation || name
     end
 
+    def country_row(degree)
+      {
+        key: t('application_form.degree.institution_country.new_review_label'),
+        value: international?(degree) ? COUNTRIES_AND_TERRITORIES[degree.institution_country] : 'United Kingdom',
```

Comment:
We don't appear to capture `institution_country` in the current flow for uk degrees. I've kept it the same in this PR so it means that I have to hardcode that any non-international degree is 'United Kingdom' for country